crates/hdk/src/capability.rs

```diff
@@ -175,15 +177,15 @@ pub fn generate_cap_secret() -> ExternResult<CapSecret> {
     })
 }
 
-/// Update a capability secret.
+/// Update a capability grant.
 ///
-/// Wraps the [ `update` ] HDK function with system type parameters set.
+/// Wraps the [`update`] HDK function with system type parameters set.
 /// This guards against updating application entries or setting the wrong entry types.
 ///
 /// Capability grant updates work exactly as a delete+create of the old+new grant entries.
 ///
-/// The first argument is the header hash of the old grant being deleted as per [ `delete_cap_grant` ].
-/// The second argument is the entry value of the new grant to create as per [ `create_cap_grant` ].
+/// The first argument is the header hash of the old grant being deleted as per [`delete_cap_grant`].
+/// The second argument is the entry value of the new grant to create as per [`create_cap_grant`].
 pub fn update_cap_grant(
     old_grant_header_hash: HeaderHash,
     new_grant_value: CapGrantEntry,
```

Comment:
I'm just realizing that this function probably also needs the ability to specify `ChainTopOrdering`...